test/payment/eip712/ApproveForwarder.eip712.js

```diff
@@ -0,0 +1,31 @@
+module.exports = function (chainId, verifyingContract, forwarder, approved = true, nonce = 0) {
```

Comment:
recommend to move this file somewhere else than in payment folder as it is only related to meta-tx